Added Section - Inactive Mailboxes and Auto-Expanding Archive (#1330)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/inactive-mailboxes-in-office-365.md

If a hold that's associated with an eDiscovery case in the Security & Compliance Center is placed on a mailbox and then the mailbox or the user's Office 365 account is deleted, the mailbox becomes an inactive mailbox. However, we don't recommend using eDiscovery case holds to make a mailbox inactive. That's because eDiscovery cases are intended for specific, time-bound cases related to a legal issue. At some point, a legal case will probably end and the holds associated with the case will be removed and the eDiscovery case will be closed. In fact, if a hold that's placed on an inactive mailbox is associated with an eDiscovery case, and then the hold is released or the eDiscovery case is closed (or deleted), the inactive mailbox will be permanently deleted. Also, you can't create a time-based eDiscovery hold. That's means that content in an inactive mailbox is retained forever or until the hold is removed and the inactive mailbox is deleted. Therefore, we recommend using a Litigation Hold or an Office 365 retention policy for inactive mailboxes.
  
For more information about eDiscovery cases and holds, see [eDiscovery cases](ediscovery-cases.md).

> ## Inactive Mailboxes and Auto-Expanding Archive

Reviewer: ```suggestion
## Inactive mailboxes and auto-expanding archive
```

> ## Inactive Mailboxes and Auto-Expanding Archive

Reviewer: ```suggestion
## Inactive mailboxes and auto-expanding archive
```

>  An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios, where it is necessary to recover data for these specific types of mailboxes it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:

Reviewer: ```suggestion
 An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios where it is necessary to recover data for these specific types of mailboxes, it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:
```

>  An inactive mailbox that is configured with an Auto-Expanding Archive cannot be recovered or restored. In scenarios, where it is necessary to recover data for these specific types of mailboxes it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location. For step-by-step procedures for searching mailboxes and exporting search results, see the following topics:

Reviewer: ```suggestion
 An inactive mailbox that is configured with an auto-expanding archive cannot be recovered or restored. In scenarios where it is necessary to recover data for these specific types of mailboxes, it is recommended that the data is exported from the mailbox using the Content Search tool and imported to the desired location.
 For step-by-step procedures to search mailboxes and export search results, please see [Content Search in Office 365](https://docs.microsoft.com/microsoft-365/compliance/content-search?view=o365-worldwide) and [Export Content Search results](https://docs.microsoft.com/microsoft-365/compliance/export-search-results?view=o365-worldwide).
```

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> Content Search in Office 365

Reviewer: ```suggestion
[Content Search in Office 365](https://docs.microsoft.com/microsoft-365/compliance/content-search?view=o365-worldwide)
[Export Content Search results](https://docs.microsoft.com/microsoft-365/compliance/export-search-results?view=o365-worldwide)
```

> Content Search in Office 365

Reviewer: ```suggestion
```

> Content Search in Office 365

Reviewer: ```suggestion
```

> Export Content Search results

Reviewer: ```suggestion
```

> Export Content Search results

Reviewer: ```suggestion
```
